File: 프로그래머스/lv0/120853. 컨트롤 제트/컨트롤 제트.py
def solution(s):
    
    ans = s.split(" ")
    for i in range(len(ans)):
        if ans[i] == 'Z':
            ans[i-1] = 0
    
    n = 0
    for i in ans:
        if i != 'Z':
            n += int(i)
    
    return n
            

File: 프로그래머스/0/120853. 컨트롤 제트/컨트롤 제트.java
import java.util.*;
class Solution {
    public int solution(String s) {
        int answer = 0;
        String[] ss = s.split(" ");
        boolean check = false;
        for(int i = ss.length - 1; i >= 0; i--) {
            
            if (check) {
                check = false;
                continue;
            }
            
            if (ss[i].equals("Z")) {
                check = true;
            } else {
                answer += Integer.parseInt(ss[i]);
            }
        }
        
        return answer;
    }
}